odoo/tools/xml_utils.py

```diff
@@ -14,3 +14,31 @@ def check_with_xsd(tree_or_str, xsd_path):
         #import UserError only here to avoid circular import statements with tools.func being imported in exceptions.py
         from odoo.exceptions import UserError
         raise UserError('\n'.join(str(e) for e in xml_errors.error_log))
+
+def xml_node_chain(first_parent_node, tags_list, str_value=None, return_node=None):
```

Comment:
method should be renamed into create_xml_node_chain(first_parent_node, nodes_list, last_node_value=None, return_node=None)